index.js

```diff
@@ -25,6 +25,7 @@ var lnCms = angular.module('lnCms', [
           var state = {
             name: route.state,
             url: route.url,
+            request: 'request' in route ? route.request : true,
```

Comment:
All the custom data that we attach to the states, should be inside the "data" object. That's the way ui-router recommends in order to avoid conflicts with the reserved options.